### PR TITLE
Dc 15 Helm Configuration

### PR DESCRIPTION
**Pull request by nibivi77**
### Description

Added Helm template configurations. Mostly copied from the [DTS SRE deployment templates](https://github.com/DTS-STN/dts-sre-deployment-templates/tree/main/kubernetes-helm-template).

Please make any changes you see fit.

### What to test for/How to test

I assume this will not function at first try. We will need to test it with DC once the repository is created as next-template repository is not being deployed.

### Additional Notes

Read through the files and add comments/edit the file as you see fit.

### CODE REVIEW

File: helm/charts/next-template/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+appVersion: "1.0"
+description: Generic Helm Chart for deploying a single tier application
+name: single-tier-application
+version: 0.0.1
+maintainers:
+  - name: Digital Technology Solutions, Site Reliability Engineering
+    email: gocmts@gmail.com
+    url: https://confluence.dts-stn.com/display/OPS/Site+Reliability+Engineering
```
Comment (DaceyTom2) [inline helm/charts/next-template/Chart.yaml:9]: Change, add or remove the maintainer to our group. We will be responsible for the template in this app so we don't want people accidentally going straight to SRE for help

File: helmfile/helmfile.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  int:
+  main:
+  sprint:
+  prod-blue:
+  prod-green: 
+
+releases:
+# Single Tier Website
+  - name: {{ requiredEnv "APP_NAME" }}-{{ .Environment.Name }}
+    chart: ../helm/Charts/single-tier-appliation
```
Comment (DaceyTom2) [inline helmfile/helmfile.yaml:18]: I'm assuming you had a talk with Larabie about this one but highlighting that we weren't 100% on the paths so it doesn't get forgotten

**Review by DaceyTom2 — CHANGES_REQUESTED** [review]
Small thing

File: helmfile/helmfile.yaml (hunk 2 above)
Comment (nibivi77) [reply helmfile/helmfile.yaml:18]: It is correct since the helm folder had to be created to keep the charts,values and templates together.

**Review by DaceyTom2 — APPROVED** [review]
Looks good from what I can tell, would be good to get some more experienced eyes on this change

**Review by ben851 — APPROVED** [review]

**Pull request merged**